wurst/objects/abilities/MagicResistance.wurst

```diff
@@ -13,20 +13,19 @@ import LocalAssets
 import StringExtensions
 import ColorUtils
 
-// Max level of magic resistance is 10 which = 100%.
 let MAX_LEVEL = 10
-// Minimum level of magic resistance is 1 which = 10%.
 let MIN_LEVEL = 1
 // Magic resist per lvl.
 let RESIST_PER_LVL = .1
 
 // List of the buffs for magic resistance ability.
-let MAGIC_RESIST_BUFFS = compiletime(map((lvl, _) -> createDummyBuff(lvl), range(MIN_LEVEL, MAX_LEVEL).toList()))
+let MAGIC_RESIST_BUFFS = compiletime(map((lvl, _) -> createDummyBuff(lvl), range(MIN_LEVEL - 1, MAX_LEVEL + 1).toList()))
 
 // Creates the buffs.
 function createDummyBuff(int lvl) returns int
-    return createDummyBuffObject(lvl.toString().color(MAGIC_RESIST_COLOR)+"0% Magic Resist".color(MAGIC_RESIST_COLOR),
-                                 "This unit is resisting "+lvl.toString()+"0% of magic damage.", LocalIcons.bTNResistMagic).abilId
+    return createDummyBuffObject("{0}0% Magic Resist".format(lvl.toString()).color(MAGIC_RESIST_COLOR),
```

Comment:
Base this on `RESIST_PER_LEVEL`, e.g. `"{0}%".format(RESISTER_PER_LEVEL * lvl * 100)`. You may need to use `real.round()` if the result looks like `89.99%` instead of `90%`, for example.